packages/loot-core/src/server/accounts/app.ts

```diff
@@ -696,13 +696,17 @@ function handleSyncError(
   err: Error | PostError | BankSyncError,
   acct: db.DbAccount,
 ): SyncError {
-  if (err instanceof BankSyncError) {
+  // TODO: refactor bank sync logic to use BankSyncError properly
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
This isn't ideal, but there's a considerable amount of work to do on bank sync in loot core, I'll defer this to then.